core/examples/generator/railjson_generator/schema/infra/route.py

```diff
@@ -19,23 +15,33 @@ class Route:
     entry_direction: Direction
     switches_group: Mapping[str, str] = field(default_factory=dict)
     tvd_sections: List[TVDSection] = field(default_factory=list)
-    label: str = field(default_factory=_route_id)
+    label: str = field(default=None)
     path_elements: List["PathElement"] = field(default_factory=list)
 
     _INDEX = 0
```

Comment:
Can be removed

Reply:
It doesn't seem so, removing it makes it a required field.